### PR TITLE
Feature/MD-2734 Worker Validates Spreadsheet

### PR DESCRIPTION
**Pull request by brianhyder**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -81,10 +81,38 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
++ Response 200 (application/json)
+    + Attributes (Base Collection Response)
+        + `data` (array[Asset Import], required)
+
+## Asset Import Create [/accounts/{accountId}/import/assets]
+This endpoint allows for an import to be created and scheduled
+
++ Parameters
+    + accountId: 2 (number, required) - {INT} The account ID the resource belongs to.
+    + file: data (file, required) - File to upload.
```
Comment (brianhyder) [inline apiary.apib:107]: Apparently, having an arbitrary type called "file" will trigger the correct type to generate

File: apiary.apib [hunk 2]
```diff
@@ -95,16 +123,51 @@ This endpoint allows asset import data to be found.
 + Response 200 (application/json; charset=utf-8)
     + Attributes (Asset Import)
 
-## Asset Export [/accounts/{accountId}/export/assets]
+## Asset Import Log Collection [/accounts/{accountId}/import/assets/{assetImportId}/logs{?page,pagesize,orderby}]
```
Comment (elizabethmidgett) [inline apiary.apib]: Do we need to be able to filter the logs by logType? (to filter and show only errors)

**Comment by elizabethmidgett** [comment]
+1

File: apiary.apib [hunk 3]
```diff
@@ -135,11 +135,11 @@ This endpoint allows asset import data to be found.
     + logType: Info (Log Type, optional) - The type of logs that you would like to retrieve
     + createdDateTimeAfter: `2016-01-01T00:00:00.000Z` (string, optional) - {DATETIME} Retrieves all of the logs after the date/time specified
 
-### POST
+### GET
```
Comment (elizabethmidgett) [inline apiary.apib:138]: How did I miss this!? Oy.

**Pull request merged**